### PR TITLE
Increase memory limit to prevent PostGIS OOMKilled

### PR DESCRIPTION
**Pull request by mkfdoherty**
When installing the PostGis extension the PostgreSQL container is OOMKilled. The minimal viable memory limit to prevent this issue is 200Mi.

# Short Description

_Please provide a brief summary of your changes_

# Details

# Note

WARNING: Only users listed in the CODEOWNERS file can approve PRs!

# Checks

- [ ] Documentation has been adjusted
- [ ] Architectural decisions have been documented
- [ ] Changelog has been updated
- [ ] PR is approved by a code owner
- [ ] Manifests are updated
- [ ] Commit message adheres to our [guideline](https://anynines.atlassian.net/wiki/spaces/DS/pages/2423193626/Version+Control+Workflow)

### CODE REVIEW

File: deploy/a8s/manifests/postgresql-images.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 data:
-  backupAgentImage: public.ecr.aws/w5n9a2g2/a9s-ds-for-k8s/dev/backup-agent:81450cbb7f94fa020441361d8c64ew89499acd13
+  backupAgentImage: public.ecr.aws/w5n9a2g2/a9s-ds-for-k8s/dev/backup-agent:781ca7b6e9bb4af66aeedb5046b50dc1c2b6c638
```
Comment (lhaendler) [inline deploy/a8s/manifests/postgresql-images.yaml:3]: How is this change related to the OOMKill?

Comment (mkfdoherty) [reply deploy/a8s/manifests/postgresql-images.yaml:3]: Should I just drop the commit?

Comment (mkfdoherty) [reply deploy/a8s/manifests/postgresql-images.yaml:3]: Do you have a suggestion?

Comment (lhaendler) [reply deploy/a8s/manifests/postgresql-images.yaml:3]: let's drop it. @abdulhaseeb3 can you look into why this change was made? you updated the images two weeks ago, and this would revert some of those changes
